alea/submitter.py

```diff
@@ -387,6 +388,31 @@ def already_done(self, i_args: dict) -> bool:
             is_done = False
         return is_done
 
+    def combined_tickets_generator(self):
+        """Get the combined submission script for the current configuration. ``self.vcombine_n_job``
```

Comment:
```suggestion
        """Get the combined submission script for the current configuration. ``self.combine_n_job``
```

Reply:
could you maybe add a small example to make clear how/why one should use the combine_n_jobs? 😊